chain/client/src/client_actor.rs

```diff
@@ -161,6 +173,9 @@ impl ClientActor {
             doomslug_timer_next_attempt: now,
             chunk_request_retry_next_attempt: now,
             sync_started: false,
+            state_parts_task_scheduler: Box::new(move |msg: StatePartsMessage| {
```

Comment:
Tbh first I just wanted to avoid making chain dependent on actix. Then it became unavoidable, but some tests can live with empty closures, so I left it as is